pkg/operator/ceph/upgrade/upgrade.go

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2019 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package upgrade
```

Comment:
From rook's perspective these are just ceph commands, right? What if we move upgrade.go to `pkg/daemon/ceph/client`?

Reply:
Indeed, they are just ceph commands, I guess I was trying to guide developers to a file that contains upgrade actions.